src/modules/fw_att_control/fw_att_control_main.cpp

```diff
@@ -1179,6 +1220,11 @@ FixedwingAttitudeControl::task_main()
 				_actuators.control[actuator_controls_s::INDEX_THROTTLE] = _manual.z;
 			}
 
+			// Add feed-forward from roll control output to yaw control output
+			// This can be used to counteract the adverse yaw effect when rolling the plane
+			_actuators.control[actuator_controls_s::INDEX_YAW] += _parameters.roll_to_yaw_ff * math::constrain(
```

Comment:
Thanks for the explanation.

Do you think this should be with the rest of the yaw controller?